src/mame/motorola/mex68kecb.cpp

```diff
@@ -104,6 +107,9 @@ static INPUT_PORTS_START( mex68kecb )
 	PORT_DIPSETTING(0x04,  "300")
 	PORT_DIPSETTING(0x02,  "150")
 	PORT_DIPSETTING(0x01,  "110")
+
+	PORT_START("ABORT")
+	PORT_BIT(0x01, IP_ACTIVE_LOW, IPT_BUTTON1) PORT_NAME("Abort button") PORT_CHANGED_MEMBER(DEVICE_SELF, mex68kecb_state, abort_button, 0)
```

Comment:
Do you want to give this a different default mapping, as I expect Control is used for typing control characters with the terminal?  You can use `PORT_CODE` to set the default assignment.

Reply:
Aha, excellent. That’ll be handy.